spec/fixtures/geonames_json_mpls

```diff
@@ -0,0 +1 @@
"lang":"post"},{"name":"55467","lang":"post"},{"name":"55470","lang":"post"},{"name":"55472","lang":"post"},{"name":"55474","lang":"post"},{"name":"55478","lang":"post"},{"name":"55479","lang":"post"},{"name":"55480","lang":"post"},{"name":"55483","lang":"post"},{"name":"55484","lang":"post"},{"name":"55485","lang":"post"},{"name":"55486","lang":"post"},{"name":"55487","lang":"post"},{"name":"55488","lang":"post"},{"name":"City of Minneapolis"},{"name":"Gakaabikaang"},{"name":"http://en.wikipedia.org/wiki/Minneapolis","lang":"link"},{"name":"http://ru.wikipedia.org/wiki/%D0%9C%D0%B8%D0%BD%D0%BD%D0%B5%D0%B0%D0%BF%D0%BE%D0%BB%D0%B8%D1%81","lang":"link"},{"name":"Mineapoli","lang":"haw"},{"name":"Mineapolis","lang":"lt"},{"name":"Mineápolis","lang":"es"},{"name":"Mineapolisa","lang":"lv"},{"name":"Minneapolis","lang":"ca"},{"name":"Minneapolis","lang":"da"},{"name":"Minneapolis","lang":"de"},{"isPreferredName":true,"name":"Minneapolis","lang":"en"},{"name":"Minneapolis","lang":"eo"},{"name":"Minneapolis","lang":"fi"},{"name":"Minneapolis","lang":"fr"},{"name":"Minneapolis","lang":"ga"},{"name":"Minneapolis","lang":"gl"},{"name":"Minneapolis","lang":"hu"},{"name":"Minneapolis","lang":"id"},{"name":"Minneapolis","lang":"io"},{"name":"Minneapolis","lang":"it"},{"name":"Minneapolis","lang":"la"},{"name":"Minneapolis","lang":"nl"},{"name":"Minneapolis","lang":"nn"},{"name":"Minneapolis","lang":"no"},{"name":"Minneapolis","lang":"pl"},{"name":"Minneapolis","lang":"pt"},{"name":"Minneapolis","lang":"sk"},{"name":"Minneapolis","lang":"sv"},{"name":"Minneapolis","lang":"tr"},{"name":"Minnéapolis","lang":"ug"},{"name":"Minneapòlis","lang":"oc"},{"name":"Mìn-nì-â-pô-li-sṳ̂","lang":"hak"},{"name":"Minyapolis","lang":"tl"},{"name":"MSP","lang":"iata"},{"name":"منیاپولس","lang":"ur"},{"name":"مینیاپولیس","lang":"fa"},{"name":"مينيابولس","lang":"ar"},{"name":"מיניאפוליס","lang":"he"},{"name":"Μινεάπολη","lang":"el"},{"name":"Горад Мінеапаліс","lang":"be"},{"name":"Минеаполис","lang":"bg"},{"name":"Минеаполис","lang":"mk"},{"name":"Минеаполис","lang":"sr"},{"name":"Миннеаполис","lang":"kk"},{"name":"Миннеаполис","lang":"ky"},{"name":"Миннеаполис","lang":"mrj"},{"name":"Миннеаполис","lang":"os"},{"name":"Миннеаполис","lang":"ru"},{"name":"Миннеаполис","lang":"sah"},{"name":"Миннеаполис","lang":"tg"},{"name":"Міннеаполіс","lang":"uk"},{"name":"Մինեապոլիս","lang":"hy"},{"name":"მინეაპოლისი","lang":"ka"},{"name":"मिनियापोलिस","lang":"hi"},{"name":"मिनीयापोलिस","lang":"mr"},{"name":"મિનેપોલિસ","lang":"gu"},{"name":"మిన్నియాపోలిస్","lang":"te"},{"name":"ಮಿನ್ನಿಯಾಪೋಲಿಸ್","lang":"kn"},{"name":"மினியாப்பொலிஸ்","lang":"ta"},{"name":"মিনিয়াপোলিস","lang":"bn"},{"name":"明尼亞波利斯","lang":"yue"},{"name":"明尼阿波利斯","lang":"lzh"},{"name":"明尼阿波利斯","lang":"wuu"},{"name":"明尼阿波利斯","lang":"zh"}],"adminName2":"Hennepin County","name":"Minneapolis","fclName":"city, village,...","countryName":"United States","fcodeName":"seat of a second-order administrative division","adminName1":"Minnesota"}
```

Comment:
👏 What a lovely town. 😉

Reply:
I take your point ... I was mimicking how BaseResource works, with the idea that there might be other future uses for this class. But for the current implementation, there aren't, so I'll nuke it.

Reply:
Yeah, I was going to change it to Sheffield, but I didn't want to upset you ;)